Detectors/FIT/FT0/simulation/src/Digits2Raw.cxx

```diff
@@ -109,7 +131,7 @@ void Digits2Raw::readDigits(const std::string& outDir, const std::string& fileDi
     digTree->GetEntry(ient);
 
     int nbc = digitsBC.size();
-    LOG(DEBUG) << "Entry " << ient << " : " << nbc << " BCs stored";
+    LOG(INFO) << "Entry " << ient << " : " << nbc << " BCs stored";
```

Comment:
Why such a level of verbosity is needed? If it is needed, perhaps you can hide it under mVerbosity option and keep it low by default?